liquidcore/home/templates/homepage.html

```diff
@@ -21,54 +21,15 @@ <h1>{{ liquid_title }}</h1>
 
 <p>Welcome to Liquid Investigations.</p>
 
-{% if hoover_app_url %}
-<p>
-  <a href="{{ hoover_app_url }}">Hoover</a> is a search app.
-</p>
-{% endif %}
 
-{% if hypothesis_app_url %}
+{% for app, values in liquid_apps.items%}
 <p>
-  <a href="{{ hypothesis_app_url }}">Hypothesis</a> is an annotation system.
-</p>
-{% endif %}
-
-{% if dokuwiki_app_url %}
-<p>
-  <a href="{{ dokuwiki_app_url }}">DokuWiki</a> is a wiki system used as a
-  knowledge base for processed information.
-</p>
-{% endif %}
-
-{% if rocketchat_app_url %}
-<p>
-  <a href="{{ rocketchat_app_url }}">Rocket.Chat</a> is the chat app.
-</p>
-{% endif %}
-
-{% if codimd_app_url %}
-<p>
-  <a href="{{ codimd_app_url }}">CodiMD</a> is a real-time collaboration pad.
-</p>
-{% endif %}
-
-{% if nextcloud_app_url %}
-<p>
-  <a href="{{ nextcloud_app_url }}/index.php/login?autologin=uploads">Nextcloud</a> has a
-  file share system and a contact list of users.
-</p>
-  {% if user.is_superuser %}
-  <p>
-    <a href="{{ nextcloud_app_url }}/index.php/login?autologin=admin">Nextcloud -- Admin</a>
-  </p>
+  {% if values.adminOnly == True and user.is_superuser or values.adminOnly == False %}
```

Comment:
Use implicit values when checking this expression: `if values.adminOnly and user.is_superuser or not values.adminOnly`